deno.json

```diff
@@ -0,0 +1,15 @@
+{
+  "fmt": {
+    "files": {
+      "exclude": [
+        "./tests/target",
+        "./target"
+      ]
+    }
+  },
+  "tasks": {
+    "fmt": "deno fmt && cargo fmt",
+    "build": "deno run -A --unstable https://raw.githubusercontent.com/denoland/wasmbuild/abd231f7f96c4df14206eb5dc30dbc4fb06d0b21/main.ts",
```

Comment:
Since we are dogfooding, Can we just use the local version? Sounds like a good 'test'

Reply:
We can't really depend on the local version because if we break it then we can no longer build to fix it 😄

I'm going to update this to use this version once we merge it to main.